test/app.test.js

```diff
@@ -1,13 +1,13 @@
 /* global it, describe, before */
 var request = require('supertest');
 var server = require('../server');
-// var should = require('chai').should();
 var expect = require('chai').expect;
 var Browser = require('zombie');
 
 Browser.localhost('localhost', 3000);
 
 describe('User visits the main page', function() {
```

Comment:
@marciodaitan can we split this test file so it only import/require the other test files ?
That way we can organize test files in a similar structure like we are doing with controllers